common/djangoapps/contentserver/caching.py

```diff
@@ -0,0 +1,47 @@
+"""
+Helper functions for caching course assets.
+"""
+from django.core.cache import get_cache
+from opaque_keys import InvalidKeyError
+
+CONTENTSERVER_VERSION=1
+
+# See if there's a "course_assets" cache configured, and if not, fallback to the default cache.
+try:
+    cache = get_cache('course_assets')
+except:
```

Comment:
We're not creating it at this very moment.  It's just futureproofing in case we do want to have it be separate.